index.php

```diff
@@ -1 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Cartoon Collection</title>
```

Comment:
might want the other meta tag in here which makes viewports standard size, and also you haven't linked to the stylesheet.

Reply:
this was fixed in commit 47d9b3c (gave the wrong commit messages)